swift/ql/lib/change-notes/2024-01-05-parameterized-cfg-library.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The control flow graph library (`codeql.swift.controlflow`) has been transitioned to use the shared implementation from the `codeql/controlflow` qlpack. No result changes are expected due to this change.
```

Comment:
I'm not sure I would have put a change note here (since no users are expected to notice this change), but I guess it doesn't hurt 😄

Reply:
Ah, that's my fault - I asked for a change note because the check was failing, without thinking about whether should have just added the flag to say that one is not required.  I agree it doesn't do any harm.